### PR TITLE
AnimationManager and WaveManager split

### PR DESCRIPTION
**Pull request by duncanspumpkin**

### CODE REVIEW

File: src/OpenLoco/openloco.vcxproj [hunk 1]
```diff
@@ -62,10 +62,12 @@
     <ClCompile Include="Localisation\Languages.cpp" />
     <ClCompile Include="Localisation\StringManager.cpp" />
     <ClCompile Include="Localisation\Unicode.cpp" />
+    <ClCompile Include="Map\AnimationManager.cpp" />
     <ClCompile Include="Map\MapGenerator.cpp" />
     <ClCompile Include="Map\SurfaceTile.cpp" />
```
Comment (AaronVanGeffen) [inline src/OpenLoco/openloco.vcxproj:67]: You've removed this file, right?

Comment (duncanspumpkin) [reply src/OpenLoco/openloco.vcxproj:67]: No I'm about to readd it for a tile update so didn't see the point

**Review by AaronVanGeffen — CHANGES_REQUESTED** [review]
Diverges on 910d195c69dfef5b4dd4245e827a9450590f4fab ("Implement Wave Manager update"). The commit before that, c3bfd2f82eec2938c918a7fb5087e3ddd02badbe ("Cleanup and implement reset") is actually fine, though, so that narrows it down.

**Review by AaronVanGeffen — APPROVED** [review]
Tested e43f8cf6a96eca6c5a62c6dc25c3cb7467b63ece (anim HEAD) against cba3bd06d1b0011f87a833524144ba4ae0ff7b8a (master HEAD) to find no divergence.

This is now good to go, as far as I'm concerned.

**Pull request merged**